go.mod

```diff
@@ -1,10 +1,41 @@
 module mosn.io/holmes
 
-go 1.14
+go 1.18
 
 require (
 	github.com/gin-gonic/gin v1.7.7
-	github.com/shirou/gopsutil v3.20.11+incompatible
-	github.com/stretchr/testify v1.7.0
-	mosn.io/pkg v0.0.0-20211217101631-d914102d1baf
+	github.com/shirou/gopsutil v3.21.11+incompatible
+	github.com/stretchr/testify v1.7.1
+	mosn.io/pkg v1.1.1-0.20221021061949-9ac98b78ab84
```

Comment:
draft until next release of pkg, which shouldn't happen until https://github.com/mosn/pkg/issues/66